signed/verifiers.go

```diff
@@ -0,0 +1,69 @@
+package signed
+
+import (
+	"crypto"
+	"crypto/rsa"
+	"crypto/sha256"
+	"crypto/x509"
+
+	"github.com/flynn/go-tuf/Godeps/_workspace/src/github.com/agl/ed25519"
+)
+
+// The Verifier interface provides flexibility over exactly how a verifier
+// is implemented vs having a simple function type.
```

Comment:
This comment should describe what a `Verifier` does instead of comparing implementation options.